Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -244,7 +244,6 @@ void TextureCacheBase::ScaleTextureCacheEntryTo(TextureCacheBase::TCacheEntry* e
   {
     return;
   }
-
```

Comment:
I must have left some lines when removing the rebase conflicts, I'll get a feel for it how it should stick together in future.